ingest-app/docs/api.md

```diff
@@ -1209,3 +1213,11 @@ curl -i -H "Echo-Token: XXXX" -H "Cmr-Pretty:true" %CMR-ENDPOINT%/granule-bulk-u
 }
 ```
 Granule bulk update tasks and statuses are available for 90 days.
+
+### Refresh Granule Bulk Update Status
+Granule bulk update task statuses can be refreshed manually, provided the user has the appropriate permissions, with the following command.
```

Comment:
Add more description on exactly what this endpoint does and specify the exact permission needed to perform this task.